src/Umbraco.Forms.Integrations.Crm.Hubspot/HubspotWorkflow.cs

```diff
@@ -36,10 +36,10 @@ public HubspotWorkflow(ILogger<HubspotWorkflow> logger, IContactService contactS
             Group = "CRM";
         }
 
-        [Setting("Field Mappings", Description = "Map Umbraco Form fields to HubSpot contact fields", View = "~/App_Plugins/UmbracoForms.Integrations/Crm/Hubspot/hubspotfields.html")]
+        [Setting("Field Mappings", Description = "Map Umbraco Form fields to HubSpot contact fields", View = "Hubspot.PropertyEditorUi.Mapping")]
         public string FieldMappings { get; set; }
 
-        public override WorkflowExecutionStatus Execute(WorkflowExecutionContext context)
+        public WorkflowExecutionStatus Execute(WorkflowExecutionContext context)
         {
             var workflowName = GetWorkflowName();
```

Comment:
Is the method `GetWorkflowName` used in this case? I can see in the comments that it was introduced between Forms 11 and 12.